src/resource_clients/actor.ts

```diff
@@ -1,72 +1,62 @@
-const { ACT_JOB_STATUSES } = require('@apify/consts');
-const ow = require('ow').default;
-const ActorVersionClient = require('./actor_version');
-const ActorVersionCollectionClient = require('./actor_version_collection');
-const BuildCollectionClient = require('./build_collection');
-const RunClient = require('./run');
-const RunCollectionClient = require('./run_collection');
-const WebhookCollectionClient = require('./webhook_collection');
-const { ResourceClient } = require('../base/resource_client');
-const {
-    pluckData,
+import { ACT_JOB_STATUSES } from '@apify/consts';
+import { AxiosRequestConfig } from 'axios';
+import ow from 'ow';
+import { ApiClientOptions } from '../base/api_client';
+import { ResourceClient } from '../base/resource_client';
+import {
+    cast,
     parseDateFields,
+    pluckData,
+    RunWebhook,
     stringifyWebhooksToBase64,
-} = require('../utils');
+} from '../utils';
+import { ActorVersion, ActorVersionClient } from './actor_version';
+import { ActorVersionCollectionClient } from './actor_version_collection';
+import { Build } from './build';
+import { BuildCollectionClient } from './build_collection';
+import RunClient from './run';
+import RunCollectionClient from './run_collection';
+import WebhookCollectionClient from './webhook_collection';
 
 /**
  * @hideconstructor
  */
-class ActorClient extends ResourceClient {
-    /**
-     * @param {ApiClientOptions} options
-     */
-    constructor(options) {
+export class ActorClient extends ResourceClient {
+    constructor(options: ApiClientOptions) {
         super({
-            resourcePath: 'acts',
             ...options,
+            resourcePath: 'acts',
```

Comment:
Why the change?

Reply:
Wrong PR to leave a review about this, but because of this.. I will need to check if the type maybe is wrong and I should use the subresource one instead
![image](https://user-images.githubusercontent.com/17960496/129878934-c94e4134-4cc5-4576-bdff-2fbe282e5ffb.png)

Reply:
And the answer is yes, I should be using the other type.. I'll fix in that PR, and rebase appropriately

Reply:
already merged the other one, so just fix it in this one pls